Makefile

```diff
@@ -23,6 +25,11 @@ build:
 	./gradlew plugin-logo:assembleRelease;
 	./gradlew plugin-overlay:assembleRelease;
 	./gradlew plugin-scalebar:assembleRelease;
+	./gradlew plugin-lifecycle:assembleRelease;
+
+.PHONY: checkApi
+checkApi:
+	./scripts/java-api-check-all.sh "$(TAG)"
```

Comment:
Would be good to add some documentation on what's the TAG stand for, and how to use it from command line interface.